glib/src/param_spec.rs

```diff
@@ -175,7 +178,7 @@ impl ParamSpec {
 
     #[doc(alias = "g_param_spec_get_name")]
     #[doc(alias = "get_name")]
-    pub fn name<'a>(&self) -> &'a str {
+    pub fn name(&self) -> &str {
```

Comment:
Sorry, I missed this part. This change is wrong and breaks existing code.

The returned string here is an interned string that is valid for the remaining runtime of the process. It is not bound by the `&self` reference.

Reply:
Solved in https://github.com/gtk-rs/gtk-rs-core/pull/728